packages/trpc/server/admin-router/router.ts

```diff
@@ -3,13 +3,16 @@ import { TRPCError } from '@trpc/server';
 import { findDocuments } from '@documenso/lib/server-only/admin/get-all-documents';
 import { updateRecipient } from '@documenso/lib/server-only/admin/update-recipient';
 import { updateUser } from '@documenso/lib/server-only/admin/update-user';
+import { deleteDocument } from '@documenso/lib/server-only/document/delete-document';
```

Comment:
We will want to either accept an optional arg for the super delete (permanent: boolean) or provide a new method to do so.

The intention with the super delete is that if we encounter spam on the platform we want to remove it for good as we also remove the user account. We would never want to soft delete someones documents as that's for them to manage.

Reply:
A new function is been added for super-delete-document
It will always hard-delete the document.
For the case of pending status, it will send canceled emails to users.